libdokan/fs.go

```diff
@@ -535,6 +535,15 @@ func (r *Root) FindFiles(fi *dokan.FileInfo, callback func(*dokan.NamedStat) err
 		ns.FileAttributes = fileAttributeNormal
 		ns.FileSize = size
 		err = callback(&ns)
+		if err != nil {
```

Comment:
Sure, but we do it in other code and it makes it less likely for refactorings to introduce errors.